lib/recorder.dart

```diff
@@ -415,13 +414,14 @@ class _Recorder extends State<Recorder> with WidgetsBindingObserver {
   void _megaUpload(path) async {
     String url = await videoUpload(path);
 
-    ModelSong uploadObject = ModelSong(
+    ModelSong uploadObject = new ModelSong(
```

Comment:
Truth be told there's no need to write the keyword "new" in Dart.

https://stackoverflow.com/questions/50091389/do-you-need-to-use-the-new-keyword-in-dart